tests/Unit/Admin/Exports/RowBuilder/OrderItemRowBuilderTest.php

```diff
@@ -0,0 +1,177 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Tests\Unit\Admin\Exports\RowBuilder;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\Admin\Exports\RowBuilder\OrderItemRowBuilder;
+use Automattic\WooCommerce\GoogleListingsAndAds\Tests\Framework\UnitTest;
+use WC_Helper_Product;
+use WC_Helper_Order;
+use WC_Helper_Coupon;
+use WC_Order_Item_Product;
+use WC_Order_Item_Coupon;
+
+/**
+ * Class OrderItemRowBuilderTest
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\Tests\Unit\Admin\Exports\RowBuilder
+ */
+class OrderItemRowBuilderTest extends UnitTest {
+	/** @var OrderItemRowBuilder $builder */
+	protected $builder;
+
+	public function setUp(): void {
```

Comment:
Nit: `setUp()` methods run before every test, while this might only need to be set up once, in which case `setUpBeforeClass()` might be a better choice.